src/main/java/uk/gov/hmcts/ccd/domain/service/aggregated/AbstractDefaultGetCaseViewOperation.java

```diff
@@ -79,6 +83,7 @@ CaseDetails getCaseDetails(String caseReference) {
             CommonField[] caseViewFields = tab.getTabFields().stream()
                 .filter(filterCaseTabFieldsBasedOnSecureData(caseDetails))
                 .map(caseTypeTabField -> CaseViewField.createFrom(caseTypeTabField, data))
+                .map(fieldProcessorService::processCaseViewField)
                 .toArray(CaseViewField[]::new);
```

Comment:
minor:
this method is becoming a bit too long. It's generally not recommended to have long sequences of chained methods because it makes the code difficult to understand. It could be useful breaking this down